samples/tutorial/Tutorial2.md

```diff
@@ -51,21 +56,21 @@ class TodoListLayoutRunner(
 
 And then create the corresponding workflow called "TodoList".
 
-Modify the rendering to return a `TodoListScreen`, we can leave everything else as the default for now:
+Modify the rendering to return a `TodoListScreen`. Modify `State` data class to contain a placeholder parameter, to make the compiler happy. We can leave everything else as the default for now:
 
 ```kotlin
 object TodoListWorkflow : StatefulWorkflow<Unit, State, Nothing, TodoListScreen>() {
 
-  data class State()
+  data class State(val placeholder: String = "")
 
   override fun initialState(
     props: Unit,
     snapshot: Snapshot?
-  ) = Unit
+  ) = State
```

Comment:
ooops

Reply:
```suggestion
  ): State = State("initial")
```

We always want to have an explicit return type specified for `initialState` otherwise it can be inferred to a narrower type than desired.